lib/consyncful/persisted_item.rb

```diff
@@ -32,6 +30,8 @@ def delete_model(id, stats)
       nil
     end
 
+    private
```

Comment:
I am not sure `delete_model` needs to be public?

Reply:
I missed this one. I moved it outside `private` when doing this PR a couple of months ago. All done.